File: CS/DXExample.Module.Web/HidePivotGridFieldsControllerWeb.cs
using DevExpress.ExpressApp.PivotChart.Web;

namespace DXExample.Module.Web {
    public class HidePivotGridFieldsControllerWeb : HidePivotGridFieldsControllerBase {
        public HidePivotGridFieldsControllerWeb() { }
        protected override void SetFieldBuider(DevExpress.ExpressApp.PivotChart.IAnalysisControl control) {
            ((AnalysisControlWeb)control).FieldBuilder = new CustomPivotFieldBuilder(control);
        }
    }
}

File: VB/DXExample.Module.Web/HidePivotGridFieldsControllerWeb.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.PivotChart.Web

Namespace DXExample.Module.Web
	Public Class HidePivotGridFieldsControllerWeb
		Inherits HidePivotGridFieldsControllerBase
		Public Sub New()
		End Sub
		Protected Overrides Sub SetFieldBuider(ByVal control As DevExpress.ExpressApp.PivotChart.IAnalysisControl)
			CType(control, AnalysisControlWeb).FieldBuilder = New CustomPivotFieldBuilder(control)
		End Sub
	End Class
End Namespace
